minimalkv/fsspecstore.py

```diff
@@ -78,10 +79,16 @@ def readable(self) -> bool:
         return self._file.readable()
 
 
-class FSSpecStore(KeyValueStore):
+class FSSpecStore(KeyValueStore, CopyMixin):
```

Comment:
i forgot to ask yesterday: Why do we add the `CopyMixin` here?

Reply:
It's just because `fsspec` supports copying, so I thought we might as well add it. It's got nothing to do with the rest of this PR though, so I'll move it to a separate one.